src/managers/SessionManager.ts

```diff
@@ -44,6 +44,15 @@ class SessionManager {
         return session.turnIndex % playerCount === me.orderIndex;
     }
 
+    public isPlayersTurn(player: Player, playerCount: int): boolean {
```

Comment:
Why not just use isMyTurn()?

Reply:
Would also fuse it with the MyTurn.... the following way:
- Add a optional parameter to isMyTurn (orderIndex).
- Make that the equality check at the end uses the optional parameter or me.index as default.
- Rename function to isTurn.